### ISSUE TITLE
meet NAN  when I use realSR dataset to train SrFlow

### ISSUE THREAD

**Issue by ccJia** [andreas128/SRFlow#25]
At the early beginning, the loss like this 
![image](https://user-images.githubusercontent.com/7822492/110098852-9bcac880-7ddb-11eb-847c-fa0819bdfe8a.png)
But after 3 k epoch, the loss meet NaN problem ,like this:
![image](https://user-images.githubusercontent.com/7822492/110099262-109e0280-7ddc-11eb-89ca-5876c7499250.png)
I use pretrain model "SRFlow_DF2K_4X.pth" as initial weights. Could you give me some suggestion to solve this problem?

**Comment by ccJia** [comment]
without pretrain model get the same problem.
![image](https://user-images.githubusercontent.com/7822492/110101808-fdd8fd00-7dde-11eb-98cd-c034a184c909.png)

**Comment by ccJia** [comment]
without pretrain model get the same problem.
![image](https://user-images.githubusercontent.com/7822492/110101879-0fbaa000-7ddf-11eb-9988-b5925f479e8c.png)
